### PR TITLE
Adding Pay with Venmo Example Integrations to GitHub

### PR DESCRIPTION
**Pull request by travellw**

### CODE REVIEW

File: pay-with-venmo/braintree-venmo-example/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM node:12
+
+RUN apt-get update && apt-get install -y build-essential
+
+ENV APP_HOME /braintree_express_example
```
Comment (xjamundx) [inline pay-with-venmo/braintree-venmo-example/Dockerfile:5]: Just wanted to confirm this path didn't need to match the `braintree-venmo-example` path you used for the folder

File: pay-with-venmo/braintree-venmo-example/app.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (xjamundx) [inline pay-with-venmo/braintree-venmo-example/app.json:1]: Is this file for heroku basically?

**Review by xjamundx — APPROVED** [review]

File: pay-with-venmo/braintree-venmo-example/app.json (hunk 2 above)
Comment (travellw) [reply pay-with-venmo/braintree-venmo-example/app.json:1]: yeap

File: pay-with-venmo/braintree-venmo-example/Dockerfile (hunk 1 above)
Comment (travellw) [reply pay-with-venmo/braintree-venmo-example/Dockerfile:5]: I didn't think so - when i ran it on my machine everything was fine w/this path

**Review by travellw — COMMENTED** [review]
#handled

**Pull request merged**